htmresearch/support/text_preprocess.py

```diff
@@ -249,7 +249,7 @@ def _filterStuff(self,
 
     if removeStrings:
       for removal in removeStrings:
-        textString = textString.replace(removal, "")
+        textString = textString.replace(removal.lower(), "")
```

Comment:
Good catch. Originally I did it thinking we need it this way since tokenize() lower cases everything. But now I see the lower casing happens after string removal.